fastlane/Fastfile

```diff
@@ -103,6 +103,7 @@ lane :build_hybrid_example do |options|
     sh("yarn example")
   end
   Dir.chdir(File.expand_path("examples/purchaseTesterTypescript/ios", get_root_folder)) do
+    sh("pod install --repo-update")
```

Comment:
I don't really know but I guess the following `pod update` wouldn't be needed in circleci (since there is no `Podfile.lock`) but might be needed if we run this locally?

Reply:
Oh good point. But yeah that's exactly right, still useful locally.